src/metrics/metrics.h

```diff
@@ -27,35 +27,62 @@
 #include <prometheus/exposer.h>
 #include <prometheus/registry.h>
 #include <prometheus/family.h>
+
+using CounterFamily = prometheus::Family<prometheus::Counter>;
+
 #endif
 
 namespace metrics {
 
+namespace master {
+
+enum Counters {
```

Comment:
This was more of a pain than I realized, as I used implicit conversion a lot more than expected. Still, I think it's better now because it prevents programmer error from implicitly passing a non-enum to the interface functions.